doc/2-interface/menu-bar.md

```diff
@@ -176,7 +176,7 @@ it's not really useful, unless you're a developer and want to use a command stre
 - **invert values**: `00` becomes `FF`, `01` becomes `FE`, `02` becomes `FD` and so on.
 
 - **flip selection**: flips the selection so it is backwards.
-- **collapse/expand amount**: allows you to specify how much to collapse/expand in the next options.
+- **collapse/expand amount**: allows you to specify how much to collapse/expand in the next commands.
```

Comment:
"command" is not a new user-friendly word.

Reply:
I've adjusted all of these.